packages/ui/src/apis/continue.ts

```diff
@@ -82,3 +82,23 @@ export const sendContinueSetPhonePasscode = async (phone: string) => {
 
   return { success: true };
 };
+
+export const verifyContinueSetEmailPasscode = async (email: string, code: string) => {
+  await api
+    .post(`${passwordlessApiPrefix}/email/verify`, {
+      json: { email, code, flow: PasscodeType.Continue },
+    })
+    .json();
+
+  return { success: true };
+};
+
+export const verifyContinueSetSmsPasscode = async (phone: string, code: string) => {
+  await api
+    .post(`${passwordlessApiPrefix}/sms/verify`, {
+      json: { phone, code, flow: PasscodeType.Continue },
+    })
+    .json();
+
+  return { success: true };
+};
```

Comment:
should merge?

Reply:
Yes, that's a legacy API of code verification. Already have a separate issue refactoring that.